providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/steps/StepDefinitions.java

```diff
@@ -81,57 +82,67 @@ public static void setup() {
     @When("a boolean flag with key {string} is evaluated with default value {string}")
     public void a_boolean_flag_with_key_boolean_flag_is_evaluated_with_default_value_false(String flagKey,
             String defaultValue) {
-        this.booleanFlagValue = client.getBooleanValue(flagKey, Boolean.valueOf(defaultValue));
+        this.booleanFlagKey = flagKey;
+        this.booleanFlagDefaultValue = Boolean.valueOf(defaultValue);
```

Comment:
I had to change the way these tests all work to be consistent with the new ones (since identical steps must have matching implementations).

Instead of keeping the resolved value as state in the class, I'm keeping the flagkey/default values in most cases.